problemsets-mat223/linearalbegra.tex

```diff
@@ -4422,12 +4428,48 @@ \section*{Inverses}
 		A=\mat{1&2&3\\4&5&6\\7&8&9}
 	\]
 	\begin{parts}[resume]
-		\item Compute $E_1A$ and $E_2A$.  How do the resulting matrices relate to row
+		\item Compute $E_1A$ and $E_2A$. How do the resulting matrices relate to row
 		operations?
+			\begin{solution}
+				$E_1A=\mat{1&2&3\\4&5&6\\9&12&15}$ and $E_2A=\mat{1&2&3\\4&5&6\\5&4&3}$.
+
+				Both of these matrices are the result of applying to $A$ the same
+				row operation that was applied to the respective $E_1$ or $E_2$. 
```

Comment:
On second reading, I agree. I've reworded that paragraph for now, but I think it might be okay to just remove that paragraph and just use the second one as the explanation. Thoughts?